pytradfri/device/__init__.py

```diff
@@ -136,48 +138,49 @@ class DeviceInfo:
     }
     ATTR_BATTERY = "9"
 
-    def __init__(self, device):
+    def __init__(self, device: Device) -> None:
         """Create object of class."""
         self._device = device
 
     @property
-    def manufacturer(self):
+    def manufacturer(self) -> str:
         """Human readable manufacturer name."""
-        return self.raw.get(DeviceInfo.ATTR_MANUFACTURER)
+        return self.raw[DeviceInfo.ATTR_MANUFACTURER]  # type: ignore
```

Comment:
It seems using variables together with `TypedDict` isn't allowed:
`pytradfri/device/__init__.py:153: error: TypedDict key must be a string literal; expected one of ("0", "1", "2", "3", "6", ...)  [misc]`

What I found when searching on this issue was the recommendation to ignore the typing error. Perhaps we should drop the use of variables and simply use `return self.raw["0"]` instead? I've added documentation of what the keys contain to the typing file.

Reply:
I think we can drop the constants if we only use them in this class.